src/ray/raylet/local_task_manager.cc

```diff
@@ -138,6 +155,54 @@ void LocalTaskManager::DispatchScheduledTasksToWorkers() {
     }
     auto &sched_cls_info = info_by_sched_cls_.at(scheduling_class);
 
+    // Apply fairness only when the total CPU requests exceed the node capacity
```

Comment:
can you add a few lines on how this scheduling work? for example (IIUC)

```
Skips scheduling classes whose number of running tasks are more than the average number of tasks per schedule class.
```

And, a qusetion: why do we count by *number of tasks* bearing CPU reqs, instead of *total number of cpus* per sched class?

Reply:
Thanks for your suggestion so much!!!
Let we check these two problems.
1. For how to check the number of CPU request, in fact, another way is each time before dispatching, we can sum number of task each scheduling class * number of cpu for this scheduling class to get the total cpu usage for `task_to_dispatch`, I think that is more robotic and the cost is we need to do that calculation (But I think that is acceptable?)
2. This is how to define fairness. Suppose task `f` requires 10 CPU and task `g` request 1 cpu, the fair is 1 `f` vs 10 `g`, or 1 `f` vs 1 `g`? What is your opinion? 
3. If the fair is 1 `f` vs 10 `g`, I think I prefer to use  `sum( number of task each scheduling class * number of cpu)` (Cost is not high) to calculate the number of cpu each scheduling class and make decision, which is also more robotic?

Reply:
1. counting (number of sched class) * (number of cpu in this sched class) sounds good, this avoids per-task ops
2. fairness depends on the specific workload. It's regrettable we have a fixed and naive policy right now. Maybe later we can define a (Local)SchedulingPolicy or so. But today let's focus on our use case. Per @stephanie-wang 's comment let's do fairness on number of tasks, not number of CPUs. But let's document it well in the code!

Reply:
I just fix them and plz have a check! Thanks!